app/views/layouts/application.html.erb

```diff
@@ -22,6 +22,13 @@
     <!-- Provides https://jqueryui.com/sortable/ and /autocomplete/-->
     <script src="https://code.jquery.com/ui/1.13.1/jquery-ui.js"></script>
 
+    <!-- Reference bootstrap-select for the modern looking multi-select dropdown list
+      Notice that we are using a beta version of this component since that's the one
+      supported with Bootstrap 5 (https://stackoverflow.com/a/67949799/446681).
+    -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.14.0-beta2/css/bootstrap-select.min.css" integrity="sha512-mR/b5Y7FRsKqrYZou7uysnOdCIJib/7r5QeJMFvLNHNhtye3xJp1TdJVPLtetkukFn227nKpXD9OjUc09lx97Q==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
Up till now, I don't think we've had any dependencies on cloudflare. Do we have a general policy about dependence on CDNs?

Reply:
@mccalluc We have plenty of CDN dependencies already on that file: https://github.com/pulibrary/pdc_describe/blob/main/app/views/layouts/application.html.erb#L11-L26

Reply:
I would like to defer to another developer for review. For me, adding a new CDN, "cloudflare" in this case, adds a new point of fragility. Maybe they are all s3 behind the curtain, or maybe not.

Separately: I don't see the additional functionality offered by this plugin as being worth the incremental cost. Checkboxes work well and don't require outside dependencies.

But this is just my opinion: If someone else feels comfortable, please approve.

Reply:
OK. I removed the third party and went with the built-in

Reply:
... but I think there's also a larger tech debt issue that I'm concerned about. I'll get my thoughts in order and put something on the agenda.